student-hub-app/src/Home/LogIn.js

```diff
@@ -0,0 +1,57 @@
+// Login.js
+
+import React, { useState } from 'react';
+
+const LogIn = () => {
+    const [formData, setFormData] = useState({
+        email: '',
+        password: '',
+    });
+
+    const handleChange = (e) => {
+        setFormData({ ...formData, [e.target.name]: e.target.value });
+    };
+
+    const handleSubmit = (e) => {
+        e.preventDefault();
+        // Perform login logic here, e.g., make an API call
+        console.log('Login Form Data:', formData);
+        // Reset form after submission
+        setFormData({
+            email: '',
+            password: '',
+        });
+    };
+
+    return (
+        <div>
+            <h2>Login</h2>
+            <form onSubmit={handleSubmit}>
+                <label>
+                    Email:
+                    <input
+                        type="email"
+                        name="email"
+                        value={formData.email}
+                        onChange={handleChange}
+                    />
```

Comment:
Are we logging in with email rather than username?

Reply:
^^. This is signup but still, should also have username. Maybe email, username, and password